### ISSUE TITLE
Annotate inherited methods

### ISSUE THREAD

**Issue by Chriscbr** [cdklabs/jsii-docgen#598]
It may be useful to the reader of an API reference if it showed which methods were inherited from a superclass (and perhaps even which class it was inherited from?)

This might require augmenting the JSON schema for methods / callables with an additional field containing inheritance information.

**Comment by vgkowski** [comment]
I would add the requirement to mask some inherited methods like the ones inherited from Construct `toString` or `isConstruct` for example. The idea is to minimize the size of the API reference.

**Comment by Chriscbr** [comment]
Good idea. Every construct in the `Constructs` section of the generated docs by definition inherits from `Construct` so those methods are pretty superfluous.

**Comment by riywo** [comment]
Does anyone have a workaround except modifying `API.md` every time...?
